README.md

```diff
@@ -1,17 +1,16 @@
-# mad-skillzzzz
+# Quiz Game (in React!)
```

Comment:
I guess you're right... 😔

Reply:
Oh, didn't mean for my readme to be included.

Reply:
But its definitely better for 'mad-skillzzzz' to be replaced now... it had a good run